modules/sdk/project-templates/project-templates-theme/build.gradle

```diff
@@ -1,5 +1,5 @@
 dependencies {
-	compileOnly group: "com.liferay", name: "com.liferay.project.templates", version: "4.5.2"
+	compileOnly group: "com.liferay", name: "com.liferay.project.templates", version: "4.5.3"
```

Comment:
hey @lawrence-lee  this version doesn't exist yet.  Brian wont be able to publish the archetypes first before publishing `com.liferay.project.templates`

Reply:
can you remove these? Brian should be able to bump this.